tests/test_expression.py

```diff
@@ -150,7 +150,7 @@ def get_context_data(
 
         self.assertEqual(
             rendered.strip(),
-            "<div>lorem</div>\n                <div>True</div>\n                <div>[{'a': 1}, {'a': 2}]</div>",
+            "<!-- _RENDERED SimpleComponent_5b8d97,a1bc3e -->\n                <div>lorem</div>\n                <div>True</div>\n                <div>[{'a': 1}, {'a': 2}]</div>",  # noqa: E501
```

Comment:
NOTE: Since the components now leave a comment when rendered, we had to add those here and there.